src/shared/model/EncryptedState.ts

```diff
@@ -13,4 +13,12 @@ export interface EncryptedState {
 	stateHandleExpiresAt?: string;
 	// Okta IDX API - Flag to determine if the user has used a passcode
 	passcodeUsed?: boolean;
+	// Okta IDX API - State of the user in the Okta determines if we can send passcodes to the user when resetting the password
+	// 0 - user doesn't exist - not currently used in any code
+	// 1 - ACTIVE - user has email + password authenticator (okta idx email verified)
+	// 2 - ACTIVE - user has only password authenticator (okta idx email not verified)
+	// 3 - ACTIVE - user has only email authenticator (okta idx email verified) - not currently used in any code
+	// 4 - user not in ACTIVE state - not currently used in any code
+	// Only including states that we actually currently set on the EncryptedState cookie
+	userState?: 1 | 2;
```

Comment:
Could it be more-readable if it used an enum or set of defined strings? or does this come from Okta?

Reply:
Have updated it to be more clear, see [`c000429` (#2889)](https://github.com/guardian/gateway/pull/2889/commits/c000429a5e375c0ddf4c60983f1a1628c4287ea1)